bookdown/03-Acquisition_Software_Settings.Rmd

```diff
@@ -7,32 +7,34 @@ bibliography: references.bib
 
 # Satellite Data Acquisition Software and Settings {#software-settings}
 
-Currently, all satellite data in **AquaSat** are obtained using the Python API
+Currently, all satellite data in AquaSat v2 are obtained using the Python API
 for Google Earth Engine (GEE) [@gorelick2017]. While the orchestration of data
-acquisition is performed by {targets}, a workflow management program for R, all
-code directly related to GEE data acquisition is written in Python.
+acquisition is performed by the {targets} workflow, all code directly related to
+GEE data acquisition is written in Python.
 
 ## {reticulate} Conda Environment
 
 RStudio [@positteam2023] is an IDE that, alongside the {reticulate} package
 [@ushey2023], facilitates integration of R and Python code within the same
-environment. In **lakeSR**, we use [a singular R
+environment. In AquaSat v2, we use [a singular R
 script](https://github.com/AquaSat/lakeSR/blob/main/python/pySetup.R) to set up
 a {reticulate} Conda environment that is invoked at the beginning of a {targets}
-run (lines 8-12 of the
-[\_targets.R](https://github.com/AquaSat/lakeSR/blob/main/_targets.R) file) to
-be sure that our Python code runs consistently.
+run to be sure that our Python code runs consistently.
 
-| Software/Py Module | version | citation                                                     |
-|---------------------|:-----------:|--------------------------------------------|
-| Python             | 3.8.18  | Python Software Foundation, [www.python.org](www.python.org) |
-| earthengine-api    | 0.1.374 | [@gorelick2023]                                              |
-| pandas             |  2.0.3  | [@thepandasdevelopmentteam2023]                              |
-| fiona              |  1.9.5  | [@gillies2023]                                               |
-| pyreadr            |  0.5.0  | [@fajardo2023]                                               |
-| pyarrow            | 13.0.0  | [@apachearrowdevelopers2023]                                 |
+| Software/Python Module | version | citation |
+|---------------------------|:-------------------------:|---------------------------|
+| Python | 3.10.13 | Python Software Foundation, [www.python.org](www.python.org) |
+| earthengine-api | 1.4.0 | @gorelick2023 |
+| pandas | 2.0.3 | @thepandasdevelopmentteam2023 |
+| pyreadr | 0.5.2 | @fajardo2023 |
+| PyYAML | 6.0.2 | The PyYAML Project, <https://github.com/yaml/pyyaml> |
+| numpy | 1.24.4 | @harris2020 |
 
-: Python and python modules and versions used in \*\*lakeSR\*\*.
+: Python and python module
+
+The script `run_targets.Rmd` includes the steps to create this environment and
+authenticate your GEE user. These steps should be run prior to running the
+pipeline to assure a smooth run of the workflow.
 
 ## Google Earth Engine Setup
```

Comment:
thought: This section feels a little orphaned to me reading it. I think this is mostly because of the beginning: `"Creation of a GEE is easy and free. Click ‘Get Started’..."`. For some reason that gives me the expectation that more will be explained. Perhaps just ~ "You can create a GEE account here: ... Here's a link that explains how to do so" or something.